weasyl/read_staff_yaml.py

```diff
@@ -0,0 +1,83 @@
+"""
+Dictionary of Weasyl staff.
+
+This module must be initialized with a call to `read_staff_yaml._init_staff_dict()`.
```

Comment:
- It’s called `_load_staff_dict`
- If it’s meant to be used by other modules, it probably shouldn’t be prefixed with an underscore
- Why not just read the file right away?